CMakeLists.txt

```diff
@@ -78,11 +78,14 @@ set_target_properties(${LIBRARY_NAME}  PROPERTIES VERSION ${${VARS_PREFIX}_VERSI
                                        SOVERSION ${${VARS_PREFIX}_VERSION}
                                        PUBLIC_HEADER "${folder_header}")
 
-target_include_directories(${LIBRARY_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>"
-                                           "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/${${VARS_PREFIX}_INSTALL_INCLUDEDIR}>"
-                                           ${YARP_INCLUDE_DIRS}
-                                           ${iDynTree_INCLUDE_DIRS}
-                                           ${wholeBodyInterface_INCLUDE_DIRS})
+target_include_directories(${LIBRARY_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include/yarpWholeBodyInterface>"
```

Comment:
Any reason to go for:
~~~
#include "yarpWbiUtil.h"
~~~ 
over 
~~~
 #include <yarpWholeBodyInterface/yarpWbiUtil.h>
~~~
I found the latter much more clean [especially for downstream code (code that uses this library)] and in line with current YARP/ICUB code practice, and I currently am using it in iDynTree. Any reason to prefer the first one? 
Consider that if we use a mixture of both in public headers, the include directories passed to the compiler of the downstream code should include both the directory in which `yarpWholeBodyInterface` is installed, and the  `yarpWholeBodyInterface` itself.